src/grubconfigs.rs

```diff
@@ -124,4 +144,15 @@ mod tests {
         assert!(td.exists("boot/efi/EFI/fedora/grub.cfg")?);
         Ok(())
     }
+    #[test]
+    fn test_write_grubenv() -> Result<()> {
+        let td = tempfile::tempdir()?;
+        let tdp = td.path();
+        std::fs::create_dir_all(tdp.join("boot/grub2"))?;
+        let td = openat::Dir::open(&tdp.join("boot"))?;
+        write_grubenv(&td)?;
+
+        assert!(td.exists("grub2/grubenv")?);
```

Comment:
```suggestion
        assert!(td.exists("boot/grub2/grubenv")?);
```

Reply:
Ah, no, you redefine td above.

Reply:
No worry, the CI failed because not found `/usr/bin/grub2-editenv` as not install `grub2-tools-minimal`